src/cc_catalog_airflow/dags/provider_api_scripts/test_metropolitan_museum_of_art.py

```diff
@@ -0,0 +1,265 @@
+import json
+import logging
+import os
+import requests
+from unittest.mock import patch, MagicMock
+import pytest
+import metropolitan_museum_of_art as mma
+
+RESOURCES = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)),
+    'tests/resources/metropolitan_museum_of_art'
+)
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.INFO
+)
+logger = logging.getLogger(__name__)
+
+
+def test_get_object_ids():
+    r = requests.Response()
+    r.status_code = 200
+    r.json = MagicMock(return_value={
+        'total': 4,
+        'objectIDs': [153, 1578, 465, 546]
+        })
+    with patch.object(
+        mma.delayed_requester,
+        'get',
+        return_value=r
+    ) as mock_get:
```

Comment:
It's not necessary to use this variable here (`mock_get`).  It's defined, but never used.
```suggestion
    ):
```